tests/e2e/scripts/gke/deploy.sh

```diff
@@ -121,6 +122,40 @@ fi
 run kubectl create -f ${YAML_FILE} --namespace "${NAMESPACE}"
 HOST=$(get_cluster_host "${NAMESPACE}")
 
+# Run in background while e2e tests are running.
+# ESPv2 is deployed in managed mode for all these e2e tests.
+# This will cause ESPv2 to rebuild the Envoy listener while lots of traffic is running through.
+function doServiceRollout() {
+  if [ "${BACKEND}" != 'bookstore' ]; then
+    echo "doServiceRollout: TODO(nareddyt): Support managed service rollout for ${BACKEND}"
+    return 0
+  fi
+
+  while true; do
+    echo 'doServiceRollout: Sleeping until next service rollout'
+    sleep 15m
+
+    local allow_cors=''
+    if (( RANDOM % 2 )); then
+      allow_cors=true
+    else
+      allow_cors=false
+    fi
+
+    echo "doServiceRollout: Setting allowCors = ${allow_cors}"
+    local tmp_file=$(mktemp)
+    cat "${SERVICE_IDL}" \
+        | jq ".\"x-google-endpoints\"[0].allowCors = $allow_cors" \
+        > "${tmp_file}"
+    mv -f "${tmp_file}" "${SERVICE_IDL}"
+
+    echo "doServiceRollout: Deploying and rolling out new config for service ${APIPROXY_SERVICE}"
+    create_service ${CREATE_SERVICE_ARGS}
+  done
+}
+
+doServiceRollout &
```

Comment:
Is it supposed by wrapped by checking "bookstore" backend?

Reply:
Hmm, is there a good reason why?

Keep in mind config manager will take up to 1 minute to detect the new rollout, so checking right after a deploy will do nothing.

Reply:
I saw the function only modified the bookstore openapi yaml?
Also, why you only kill it if it is bookstore in line 177?

Reply:
Done, moved the if statement to clarify.